index.js

```diff
@@ -7,19 +7,35 @@ var ProviderSubprovider = require("web3-provider-engine/subproviders/provider.js
 var Web3 = require("web3");
 var Transaction = require('ethereumjs-tx');
 
-function HDWalletProvider(mnemonic, provider_url, address_index=0, num_addresses=1) {
-  this.mnemonic = mnemonic;
-  this.hdwallet = hdkey.fromMasterSeed(bip39.mnemonicToSeed(mnemonic));
+function HDWalletProvider(mnemonics, provider_url, address_index=0, num_addresses=1) {
+  if (!Array.isArray(mnemonics)) {
+    mnemonics = [mnemonics];
+  }
+  
+  this.hdwallets = [];
```

Comment:
you changed the class property `hdwallet` to `hdwallets`. This is a breaking change for all modules that use this property  (I guess someone is using it, must be a reason why they put in on `this`).

Reply:
Not sure if this is such an issue.
Yes this is breaking change, but I don't think someone is using this attributes.
example, last PR that approved also change this attributes:
[#5](https://github.com/trufflesuite/truffle-hdwallet-provider/pull/5)